server/routes/config/ConfigRoute.ts

```diff
@@ -26,8 +26,7 @@ export const ConfigRoute = (config: ServerConfig) =>
       APP_BASE: config.SERVER.APP_BASE,
     };
 
-    res.type('application/javascript').send(`
-      window.wire = window.wire || {};
-      window.wire.env = ${JSON.stringify(clientConfig)};
-    `);
+    res
```

Comment:
How about?

```
const payload = `window.wire = window.wire || {};
window.wire.env = ${JSON.stringify(clientConfig)};`

res.send(payload);
```

I feel it's slightly easier to read

Reply:
The `\n` makes things slightly hard to read

Reply:
It would then look like this:
```ts
    res
      .type('application/javascript')
      .send(`window.wire = window.wire || {};
window.wire.env = ${JSON.stringify(clientConfig)};`);
```

Reply:
But yeah, the `\n` is annoying...